assets/training/finetune_acft_common/src/validation/validation.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""script to update validation info."""
+
+
+def main():
+    """Script which runs as part of validation component to update output."""
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--validation-info", required=True, help="Model source ")
+
+    args = parser.parse_args()
+
+    print("Validation info: ", args.validation_info)
```

Comment:
Should we be printing the content of the file ?
Should we append "Validation Completed" string to file instead of overwriting ?
I am assuming validation details to be present in this file. Ignore this comment if this assumption is incorrect.